examples/custom_font_style/src/main.rs

```diff
@@ -36,7 +36,7 @@ fn content(ui: &mut egui::Ui) {
     ui.label(LOREM_IPSUM);
     ui.add_space(15.);
     ui.label(RichText::new("Sub Heading").text_style(heading2()).strong());
-    ui.label(LOREM_IPSUM);
+    ui.label(RichText::new(LOREM_IPSUM).text_style(TextStyle::Monospace));
```

Comment:
Maybe `ui.monospace(LOREM_IPSUM)` would be a bit cleaner?

Reply:
👍  changed